agent/ecscni/types.go

```
@@ -0,0 +1,89 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ecscni
+
+import (
+	"github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/cni/pkg/types"
+)
+
+const (
+	CNIPath             = "/ecs/cni"
+	CNIVersion          = "0.3.0"
+	VersionCommand      = "--version"
+	defaultEthName      = "eth0"
```

There should be no `defaultEthName`. If unspecified, we should return an error.